src/components/FeatureSelector/Defaults.tsx

```diff
@@ -62,7 +68,8 @@ const options: FeatureSliderData[] = [
     max: 100,
     currentMin: 0,
     currentMax: 100,
-    units: '%'
+    units: '%',
+    description: 'Valence describes the musical positiveness conveyed by a track'
```

Comment:
```suggestion
    description: 'Valence describes the musical/emotional positiveness conveyed by a track'
```
Our less musically inclined users might not understand what musical positiveness is